extensions/mongoProcessor/mongoProcessorTask.js

```diff
@@ -16,18 +19,82 @@ const healthServer = new HealthProbeServer({
     port: config.healthcheckServer.port,
 });
 
-const mongoQueueProcessor = new MongoQueueProcessor(kafkaConfig,
-    mongoProcessorConfig, mongoClientConfig);
-
+const log = new werelogs.Logger('Backbeat:MongoProcessor:task');
 werelogs.configure({ level: config.log.logLevel,
     dump: config.log.dumpLevel });
 
-mongoQueueProcessor.start();
-healthServer.onReadyCheck(log => {
-    if (mongoQueueProcessor.isReady()) {
-        return true;
-    }
-    log.error('MongoQueueProcessor is not ready!');
-    return false;
-});
-healthServer.start();
+const activeProcessors = {};
+
+function initAndStart() {
+    // NOTE: using replication service account
+    const sourceConfig = config.extensions.replication.source;
+    initManagement({
+        serviceName: 'replication',
+        serviceAccount: sourceConfig.auth.account,
+    }, error => {
+        if (error) {
+            log.error('could not load management db', { error });
+            setTimeout(initAndStart, 5000);
+            return;
+        }
+        log.info('management init done');
+
+        let bootstrapList = config.getBootstrapList();
+        config.on('bootstrap-list-update', () => {
+            bootstrapList = config.getBootstrapList();
+
+            const active = Object.keys(activeProcessors);
+            const update = bootstrapList.map(i => i.site);
+            const allSites = [...new Set(active.concat(update))];
+
+            async.each(allSites, (site, next) => {
+                if (update.includes(site)) {
```

Comment:
Minor: for readability, you can avoid the `if...else` and `if` nesting by first checking if the site is not in the updated list.